container/Containerfile

```diff
@@ -22,6 +22,10 @@ ARG OSD_FLAVOR="default"
 # (optional) Should be 'true' for CI builds (pull from shaman, etc.)
 ARG CI_CONTAINER="true"
 
+# creds for accessing prerelease packages on download.ceph.com for release builds
+ARG PRERELEASE_USERNAME ""
+ARG PRERELEASE_PASSWORD ""
```

Comment:
`ARG`s are stored in the built image, so this seems like maybe not what we want

Reply:
indeed it is not.  I have seen the secrets light and will fix

Reply:
best I got is --secret=id=<id>src=<localpath>, so I'm trying to carefully manage a local temporary file